### PR TITLE
Add Xonsh to project list

### PR DESCRIPTION
**Pull request by Carreau**
cc @scopatz @gforsyth

### CODE REVIEW

**Comment by gforsyth** [comment]
Looks good to me.

**Comment by scopatz** [comment]
Looks good to me, but @adqm and @melund should probably be aware / agree.

**Comment by Carreau** [comment]
Of course. We might want to ping the list as well. 
You can be made member of this org as well if you like, but I don't want to spam any of you more than you already are.

**Comment by scopatz** [comment]
Thanks for the offer @Carreau, but I will bow out of this one, unless you need the help

**Comment by gforsyth** [comment]
Same for me, @Carreau -- if you need a hand, I'm happy to help out where I can, but otherwise I'm good on the outside.

**Comment by melund** [comment]
I am totally with this :)

File: _sections/30-projects.md [hunk 1]
```diff
@@ -18,4 +18,5 @@ style: center
 - [QIIME](http://qiime.org)
 - [gala](https://gala.readthedocs.io)
 - [cual-id](https://github.com/johnchase/cual-id)
+- [xonsh](https://xon.sh)
```
Comment (takluyver) [inline _sections/30-projects.md]: xonsh can go quite a bit higher, using the prominence-as-indicated-by-github-stars metric: it's between sympy and scikit-bio.

**Comment by takluyver** [comment]
Thanks all. There's no particular need for extra maintainers at the moment, so no pressure.

**Comment by Carreau** [comment]
Moved up.

**Comment by scopatz** [comment]
Woohoo!

**Comment by melund** [comment]
@Carreau Cool. It seems you only moved it up on one of the lists. So xonsh now appear at a different position in the timeline and project list.

**Comment by Carreau** [comment]
> @Carreau Cool. It seems you only moved it up on one of the lists. So xonsh now appear at a different position in the timeline and project list.

Moved up.

**Pull request merged**